src/views/About.vue

```diff
@@ -1,5 +1,69 @@
 <template>
-  <div class="about">
-    <h1>Dit is de over LIWO pagina</h1>
+  <div class="about site-main">
+    <div>
+      <section class="section">
+        <h2 data-class="section-head">Tips voor sitebezoek</h2>
+        <p>
+          De informatie op deze website is ondergebracht in 'Kaarten'
+          en 'Gereedschappen'.
+        </p>
+        <p>Het kopje 'Kaarten' is voor iedereen
+      toegankelijk en bevat kaartlagen voor professionals van
+      veiligheidsregio's, waterschappen en Rijkwaterstaat, die nodig
+      zijn voor de voorbereiding op watercrises en overstromingen in
+      Nederland. Het gaat hierbij om crisisbeheersing maar het is ook
+      bruikbaar voor ruimtelijke adaptatie en versterking van
+      zelfredzaamheid.</p><p>Het kopje 'Gereedschappen' is
+      toegankelijk na inloggen met uw gebruikersaccount. Met de
+      gereedschappen kunt u als beheerder inzicht geven in de
+      status/beoordeling van uw areaal.</p><p>U kunt een account
+      aanvragen als u behoort tot de doelgroep van LIWO. Kies hiervoor
+      'Inloggen' en klik op 'Aanvragen'.</p><p>Voor gedetailleerde
+      informatie over het gebruik van deze website verwijzen we u naar
+      de gebruikershandleiding. De handleiding vindt u op
+      de <a href="https://www.helpdeskwater.nl/onderwerpen/applicaties-modellen/applicaties-per/watermanagement/watermanagement/liwo/"
+      target="_blank">LIWO pagina van de helpdesk water</a>.
```

Comment:
always add `rel="nofollow noopener"` to `target="_blank"` links
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/